### PR TITLE
add: new blog post

### PR DESCRIPTION
**Pull request by ajstrongdev**

### CODE REVIEW

File: news-13.html [hunk 1]
```diff
@@ -65,7 +65,7 @@ <h2 class="text-off-white text-3xl text-center mt-4">
...
+    Before we get into information about this release, we have a brief update about our organizational structure. As we mentioned <a href="news-11.html" class="text-rhino-purple">a while ago</a>, developer burnout hit a breaking point for some us, and in consequence development has somewhat stagnated. We have begun discussions about how we wish to structure Rhino Linux going forward, and are happy to announce that we have currently drafted a working Rhino Linux constitution, with a hard focus on community involvement. These structural changes will begin to come into effect shortly after this release, so stay tuned for the full announcement!
```
Comment (ajstrongdev) [inline news-13.html:68]: Typo in last post

File: news-14.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  </h1>
+  <p class="text-off-white mt-4">
+    After a lot of discussions with the Rhino Linux team, we are finally ready to announce changes to the way we develop. We have decided to focus on the following areas: <br>
+    &bull; Providing a strong contributor's code of conduct <br>
+    &bull; Focusing effort on strong development and user documentation <br>
+    &bull; Rewriting our website in React, which will provide us with better tools for Documentation. <br>
+    &bull; Streamlining development channels. <br>
+  </p>
+  <p class="text-off-white mt-4">
+    With our contributor's code of conduct, we aim to provide a safe and welcoming environment for all contributors. We want to ensure that everyone feels welcome and that their contributions are valued. We believe that this will help us to attract more contributors and improve the quality of our software. Our newly written contributor's code of conduct can be found <a href="https://cryptpad.fr/pad/#/2/pad/view/EeXXPAEywHh9hPe8+xfDOCPChQNvi5mqYerL+gAISyU/" class="text-rhino-purple" target="_blank">here</a>.
```
Comment (oklopfer) [inline news-14.html:69]: it should NOT be hosted on crypt pad, it should have its own page here

File: news-14.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  </p>
+  <p class="text-off-white mt-4">
+    Our decision to rewrite our website in React is going to provide us with significantly better tools when it comes to managing our documentation, publishing blog posts and ensuring long-term maintainability. We plan to write development documentation while we undertake this website redesign. This development documentation will be made publicly available to all users and contributors. By documenting our development process, we aim to provide a clear, transparent view to how Rhino Linux is maintained, and help new contributors get up to speed quickly.
```
Comment (oklopfer) [inline news-14.html:72]: do people really need to know what language we are rewriting our website in? seems like we should just announce that the website is going to get a significant rework.

File: news-14.html [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  </p>
+  <p class="text-off-white mt-4">
+    We currently do not have any plans to move development discussions away from Discord at the moment. Discord has been useful as a small team to collaborate on development and communicate effectively. We plan to now utilise a public-facing Discord forum to allow for users and contributors to discuss and organise development around specific feature requests and bug reports.
```
Comment (oklopfer) [inline news-14.html:75]: this is untrue. we have plans for forums on the new website.

File: news-14.html [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    </div>
+</div>
+
+<div class="container w-[95vw] lg:w-[60vw] mt-4 rounded-[0.65em] p-4 m-auto shadow-2xl bg-site-300">
+  <h1 class="text-center text-off-white text-3xl">
+    An update on Rhino Linux's development goals and structure
+  </h1>
+  <p class="text-off-white mt-4">
+    After a lot of discussions with the Rhino Linux team, we are finally ready to announce changes to the way we develop. We have decided to focus on the following areas: <br>
+    &bull; Providing a strong contributor's code of conduct <br>
```
Comment (oklopfer) [inline news-14.html:63]: CCC should all be capitalized

**Review by oklopfer — CHANGES_REQUESTED** [review]

**Review by oklopfer — CHANGES_REQUESTED** [review]
mission statement?

**Review by oklopfer — CHANGES_REQUESTED** [review]
mutability should be at the top, not the bottom for the CCC

File: news-14.html (hunk 3 above)
Comment (ajstrongdev) [reply news-14.html:72]: We can do, however I do not think it's necessarily a problem to mention it either?

File: news-14.html (hunk 4 above)
Comment (ajstrongdev) [reply news-14.html:75]: My plan would be that the forums would release after the new website. The forums would require significant work that will take much longer than this new website. Hence why I said "at the moment", however I can reword it to include the plans to shift to forums eventually as well.

Comment (oklopfer) [reply news-14.html:75]: > Hence why I said "at the moment", however I can reword it to include the plans to shift to forums eventually as well.
so.... we do have plans to move development discussions away from Discord at the moment. We just are not able to yet.

**Comment by ajstrongdev** [comment]
Todo:

Add mission statement

**Comment by oklopfer** [comment]
Where did mutability go

**Review by oklopfer — APPROVED** [review]
